examples/basic-tf-example/porter.yaml

```diff
@@ -12,6 +12,12 @@ parameters:
     path: /cnab/app/terraform/terraform.tfstate
     source:
       output: tfstate
+    applyTo:
```

Comment:
Added these while https://github.com/deislabs/porter/pull/1190 remains in flight...

Reply:
Removed in most recent commit, now that 1190 has been merged.